android/src/com/mapswithme/maps/settings/SettingsActivity.java

```diff
@@ -100,6 +102,12 @@ else if (header.id == R.id.about)
       Statistics.INSTANCE.trackEvent(Statistics.EventName.Settings.ABOUT);
       AlohaHelper.logClick(AlohaHelper.Settings.ABOUT);
     }
+    else if (header.id == R.id.osm_profile)
+    {
+      Statistics.INSTANCE.trackEvent(Statistics.EventName.Settings.OSM_PROFILE);
+      startActivity(new Intent(this, ProfileActivity.class));
+      return;
```

Comment:
Да, действительно.

> On Mar 21, 2016, at 5:07 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/settings/SettingsActivity.java <https://github.com/mapsme/omim/pull/2377#discussion_r56826754>:
> 
> > @@ -100,6 +102,12 @@ else if (header.id == R.id.about)
> >        Statistics.INSTANCE.trackEvent(Statistics.EventName.Settings.ABOUT);
> >        AlohaHelper.logClick(AlohaHelper.Settings.ABOUT);
> >      }
> > +    else if (header.id == R.id.osm_profile)
> > +    {
> > +      Statistics.INSTANCE.trackEvent(Statistics.EventName.Settings.OSM_PROFILE);
> > +      startActivity(new Intent(this, ProfileActivity.class));
> > +      return;
> Мне кажется, return не нужен.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2377/files/1eb3d20c55a5aa62bbe178ecfe9f3be685052a60..ea7fb378c03be65ce279f953d063c3318a0b845d#r56826754>